app/scripts/modules/core/src/application/config/DeleteApplicationSection.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import { Application, ApplicationWriter } from 'core/application';
+import { ReactInjector } from 'core/reactShims';
+import { FirewallLabel } from 'core/securityGroup/label';
+
+export interface IDeleteApplicationSection {
+  application: Application;
+}
+
+export function DeleteApplicationSection(props: IDeleteApplicationSection) {
+  const { application } = props;
+
+  function deleteApplication(): void {
```

Comment:
nit: You can use an arrow function here.

Reply:
👍